scripts/install/provision-spinnaker.md

```diff
@@ -9,12 +9,23 @@ Select the project in which you'll install Spinnaker, then click **Start**, belo
 
 ## Spinnaker Installation
 
-Now let's provision Spinnaker within {{project-id}}.
+Click the copy/paste button on the commands below to copy each command to your shell,
```

Comment:
Why not suggest they use the "Copy to Cloud Shell" button instead of the "Copy to Clipboard button".  To me they're both copy/paste buttons so this is a little confusing.

Reply:
Done.